src/app/public/modules/datepicker/datepicker-input.directive.ts

```diff
@@ -451,6 +457,14 @@ export class SkyDatepickerInputDirective
         }
       };
     }
+
+    if (this.datepickerService.isDateDisabled(dateValue)) {
```

Comment:
Presumably this works on the assumption that whatever custom dates for the date range where `dateValue` would fall has already been loaded by `datepickerService`, but what if the user inputs a disabled date without ever invoking the calendar? I think we're always going to have do asynchronously validate dates by passing it to the consumer just like we do to get the custom dates.

Reply:
Decided to defer disabled date validation to the consumer, as we don't have enough information to make those decisions ourselves.